serving/turbine_serving/llm/entrypoints/api_server.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2024 Advanced Micro Devices, Inc
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from typing import Sequence
+
+import argparse
+
+from fastapi import FastAPI, Request
+from fastapi.responses import JSONResponse, Response
+import sys
+import uvicorn
+
+app = FastAPI()
+
+
+@app.get("/health")
+async def health() -> Response:
+    return Response(status_code=200)
+
```

Comment:
Just confirming that this is as a base/reference implementation since this has no SHARK in here?

Reply:
Yes

Reply:
Awesome possum!